client/app/filters/profile-image.js

```diff
@@ -0,0 +1,9 @@
+import defaultProfileImageUrl from '@/assets/images/avatar.svg';
+
+function getUserProfileImageUrl(user) {
+  return user.is_disabled ? defaultProfileImageUrl : user.profile_image_url;
+}
+
+export default function init(ngModule) {
+  ngModule.filter('profileImage', () => getUserProfileImageUrl);
```

Comment:
Instead of having a filter and then remembering to use it everywhere, why not return the right URL in the API?